sklearn/metrics/tests/test_classification.py

```diff
@@ -1394,8 +1394,8 @@ def test_log_loss():
     assert_raise_message(ValueError, error_str, log_loss, y_true, y_pred)
 
     y_pred = [[0.2, 0.7], [0.6, 0.5], [0.2, 0.3]]
-    error_str = ('Found arrays with inconsistent numbers of '
-                 'samples: [2 3]')
+    error_str = ('Found inconsistent number of samples in input variables: '
```

Comment:
+1 for numbers of samples. I don't link "arrays" because it can also be lists, pandas dataframes, series or scipy sparse matrices.